### DISCUSSION TITLE
Deploying a Rollup App with Vercel?

### DISCUSSION THREAD

**Discussion by chpsterz** [vercel/vercel#8709] [Help]
Hello, I am trying to deploy my app with rollup to this site https://miner-flax.vercel.app/ from my github repo here https://github.com/ChpssCode/Miner

I have searched far and wide but have not found a suitable answer, any thoughts on how to get my deployment working?

**Comment by amyegan** [comment]
Hi @ChpssCode! Your site seems to be loading okay. I hope that means you already found a solution to the problem! 😁 

If you're still having trouble, please share what steps I should take to see the error and what I should expect to see. I would be happy to take a closer look.
